src/styles.scss

```diff
@@ -58,3 +58,9 @@ p {
     width: 100%;
   }
 }
+
+.title-event {
```

Comment:
This shouldn't be places in the styles file. This files is used for global styles, but this one is only used in the events component.